index.html

```diff
@@ -15,9 +16,44 @@
   </head>
 
   <body>
-    <div class="container">
-      <!-- Replace me with your html -->
+  <div class="container">
+    <p class="logoOne">Relaxr</p>
+    <p class="message">Get a piece of mind with a single tap.</p>
+    <div class="centerButton">
+      <button class="getIt">Get it now!</button>
+    </div>
+  </div>
+  <section class="greyBackground">
+    <div class="productDescription">
+      <p class="benefits">Benefits</p>
+      <p class="aboutRelaxr">The perfect personal assistant. Relaxr does your job </br> for you so you can enjoy life as 
+       it is meant to be.</p>
+      <ul class="listStyle">
+        <li>Schedule meetings for you</li>
+        <li>Excel automation to complete your</br> work for you</li>
+        <li>Responds to emails on your behalf</li>
+        <li>Does all your work for you with our</br> revolutionary AI technology</li>
+      </ul>
+    </div>
+  </section>
+  <div class="quote">
+    <p class="testimonial">"Relaxr changed my life. I’ve been able</br> to travel the world, spend 
```

Comment:
Rather than forcing line breaks with `br` tags, instead set a `width` (or `max-width`) on the element. That way, your text will be forced to wrap to a new line.  For example:

```css
.testimonial {
    color: lightgrey;
    font-size: 18px;
    font-style: italic;
    font-weight: lighter;
    text-align: center;
    max-width: 300px;  /* this will force content to wrap to a new line */
    margin: auto; /* this will center the p tag since its a block element AND we set a width */
```